src/serve.ts

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs';
```

Comment:
_could_ also do `import fs from 'fs/promises` but no biggy

Reply:
This didn't work when I ran the tests and I didn't look too deeply into why. 
When I run the tests I get this error
```
$ npm test

> @curveball/static@0.0.0 test /home/humiston/Development/static
> make lint test

node_modules/.bin/eslint --quiet 'src/*.ts' 'test/*.ts'
node_modules/.bin/nyc node_modules/.bin/mocha

Error: Cannot find module 'fs/promises'
```

And when I try running it with `yarn link` in the `a12-server`, I get a similar error.
```
$ npm run start

> a12n-server@0.12.7 start /home/humiston/Development/a12n-server
> node ./dist/app.js

internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'fs/promises'
```